build/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent none
+
+    environment {
+        MATTERMOST_DESKTOP_WINDOWS_PFX_PASSWORD = credentials('mattermost-desktop-windows-pfx-password')
+    }
+    stages {
+        stage('Build Linux') {
+            agent any
+            steps {
+                echo 'Building Mattermost Desktop App'
+                sh '''#!/bin/bash
+                  rm -rf release
+                  yarn install
+                  npm run build
+                '''
+
+                echo 'Packaging for Linux'
+                sh 'npm run package:linux'
+
+                archiveArtifacts artifacts: 'release/mattermost-desktop-*-ia32.tar.gz,release/mattermost-desktop-*[^ia32].tar.gz,release/mattermost-desktop_*_i386.deb,release/mattermost-desktop_*_amd64.deb'
+            }
+        }
+        stage('Build Windows') {
+            agent any
```

Comment:
Just curious, can we use Windows slave in future? We would be able to use Node's native modules.